net-libs/libwebsockets/libwebsockets-2.1.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="canonical libwebsockets.org websocket library"
+HOMEPAGE="https://libwebsockets.org/"
+SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
If `~x86` is added here, `repoman` will report something like:
```
  dependency.bad [fatal]        24
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: DEPEND: ~x86(default/linux/x86/13.0)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: RDEPEND: ~x86(default/linux/x86/13.0)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: DEPEND: ~x86(default/linux/x86/13.0/desktop)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: RDEPEND: ~x86(default/linux/x86/13.0/desktop)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: DEPEND: ~x86(default/linux/x86/13.0/desktop/gnome)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: RDEPEND: ~x86(default/linux/x86/13.0/desktop/gnome)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: DEPEND: ~x86(default/linux/x86/13.0/desktop/gnome/systemd)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: RDEPEND: ~x86(default/linux/x86/13.0/desktop/gnome/systemd)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: DEPEND: ~x86(default/linux/x86/13.0/desktop/kde)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: RDEPEND: ~x86(default/linux/x86/13.0/desktop/kde)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: DEPEND: ~x86(default/linux/x86/13.0/desktop/kde/systemd)
['net-libs/libhubbub']
   net-libs/libwebsockets/libwebsockets-2.1.1.ebuild: RDEPEND: ~x86(default/linux/x86/13.0/desktop/kde/systemd)
['net-libs/libhubbub']
  ebuild.badheader              1
```
I don't know why (or whether) `~86` hurts `net-libs/libhubbub` and was unable to find a workable example from ebuild files in `/usr/portage/`, so I removed it in order to bypass the warning (and get further help here from you :)